src/jest/JestFileReporter.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-useless-escape */
+import fs from 'fs';
+import JestFileReporter from './JestFileReporter';
+
+jest.mock('fs');
+jest.mock('path');
+let fsAppendSpy;
+describe('Jest File Reporter Testing', () => {
+  afterEach(() => {
+    fsAppendSpy.mockClear();
+  });
```

Comment:
want to add spaces between each test hook?
```suggestion
  });
  
```

Reply:
https://github.com/cerner/terra-toolkit-boneyard/blob/create-jest-reporter/tests/jest/TerraVerboseReporter.test.js#L14